README.md

```diff
@@ -6,125 +6,160 @@
 + Use `this` to refactor jQuery event handlers
 + Explain how `this` refers to different objects based on scope
 
-## Intro
+## Introduction
 
-We're very familiar with `self` in Ruby, and how to use self to refer to the current object when we're manipulating that object.
+It's often important in programming to know what the "owner" of a function is, so that we can operate on some specific object or data safely.
 
-For example, given a class Dog, with `name` and `owner` attributes:
+For instance, consider an event handler that fires when someone clicks
+on a link in the DOM. When that event handling function is invoked,
+we might want to know *which specific* link was clicked, so that we can
+manipulate it in some way.
 
-```ruby
-class Dog
+That's where the `this` keyword comes into play. Every function is
+automatically assigned a `this` value when called, and that represents
+"who" called the function. This value can be an object, an event, or
+even another function.
 
-  attr_accessor :name, :owner
+The value of `this` is determined by the *execution context*, or scope, of the
+function call.
 
-  def initialize(name)
-    @name = name
-  end
+## Global Scope
 
-  def bark
-    "Woof!"
-  end
+Code running outside of a function is in the **Global Scope**. Every JavaScript runtime has a default global object that will be the value of `this` when a function is called in global scope. In the browser, if we're referencing `this` in the global scope, we're referencing the `Window` object.
```

Comment:
Nit: students might be confused by capitalized `Window` vs. `window` in the browser console.